### PR TITLE
Upgrade to JoyPixels 6.5

### LINKED ISSUE #2: Updating to JoyPixels 6.5
#1 
JoyPixels has updated to a new version 6.5, as of January 26, 2021. I have created a pull request and please have a look.

### PR DESCRIPTION
**Pull request by lzcapp**

### CODE REVIEW

File: module.prop [hunk 1]
```diff
@@ -1,6 +1,6 @@
 id=emoji-joypixels
 name=JoyPixels®
-version=v6.1
-versionCode=2
-author=shivangswain
-description=Systemlessly replaces emoji font with JoyPixels 6.1.
+version=v6.5
+versionCode=3
+author=shivangswain, RainySummer
```
Comment (shivangswain) [inline module.prop]: I request you to remove your name from the author tag in the module.prop file. I'd rather add your name to the top of the README under a contributors section.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Free License
+
+Version 6.5
+
+1. Free license applies to all free emoji icons and zips.
+2. Intended for personal use, including students and teachers.
+3. Businesses can use our free emoji for testing purposes only.
+4. Upgrade to a premium license for expanded business use of free emoji.
+5. Attribution not required.
```
Comment (shivangswain) [inline README.md]: Since this PR is for upgrading to JoyPixels 6.5, I'd request you to drop the README in this PR.

**Review by shivangswain — CHANGES_REQUESTED** [review]
Thank you so much for your valuable contribution. It is appreciated. I would request you to change the following to conform to standard practices:
- Please raise an issue before requesting a PR and linking the latter to the issue.
- We follow a policy of Atomic PRs (one PR, one concern). Since this PR is for upgrading to JoyPixels 6.5, I'd request you to drop the README in this PR.
- I request you to remove your name from the author tag in the module.prop file. I'd rather add your name to the top of the README under a contributor's section.

**Comment by lzcapp** [comment]
I've remove my name from **module.prop**, deleted the **README**, and created a issue related to this pull request.

Thank you!

**Review by shivangswain — COMMENTED** [review]
Looks great. Thank you for your contribution!

**Pull request merged**